devops/docker-compose.local.yml

```diff
@@ -47,34 +66,17 @@ services:
     networks:
       - geoapi
 
-
   workers:
-    image: taccaci/geoapi-workers:local
-    networks:
-      - geoapi
-    volumes:
-      - ../:/app
-      - assets:/assets
-    environment:
-      - MAPILLARY_CLIENT_TOKEN
-      - FLASK_APP=/app/geoapi/app.py
-      - APP_ENV=local
-      - ASSETS_BASE_DIR=/assets
-      - DESIGNSAFE_URL=${DESIGNSAFE_URL}
-
-    stdin_open: true
-    tty: true
+    <<: *worker-base
```

Comment:
Cool! New syntax we're using, should we link to documentation in a comment?
https://docs.docker.com/reference/compose-file/fragments/

Reply:
good idea.  will add here and on core-portal-deployments before merging.